app/src/main/java/org/greenstand/android/TreeTracker/capture/TreeCaptureScreen.kt

```diff
@@ -48,7 +48,7 @@ fun TreeCaptureScreen(
             ActionBar(
                 leftAction = {
                     ArrowButton(isLeft = true) {
-                        navController.popBackStack()
+                        navController.navigate(NavRoute.Dashboard.route)
```

Comment:
This navigates back to the dashboard, but if I press 'back' on the dashboard I go to the camera screen again.

Reply:
Okay fixed this in a new commit.